src/pkg/backup/details/sharepoint.go

```diff
@@ -120,3 +125,28 @@ func (i *SharePointInfo) updateFolder(f *FolderInfo) error {
 
 	return clues.New("unsupported non-SharePoint ItemType").With("item_type", i.ItemType)
 }
+
+func getSiteURL(itemWebURL string) string {
```

Comment:
This looks very similar to some other code we have in the `/m365/api` package.  Can you please find a central location for it that deduplicates handling?  You can make that change in a separate PR.  Just wanted it noted here.

Reply:
found it [here](https://github.com/alcionai/corso/blob/e3363aaa46ca57c5081fd425da3c071d37d95173/src/pkg/services/m365/api/sites.go#L172-L191), but I guess it is not exactly what I am trying to do here.

Reply:
hmm, fair.  I think you could still unify the two.  But it's a low priority.